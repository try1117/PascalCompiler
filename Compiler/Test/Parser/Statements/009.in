program test;
var
  a: array[-100..100] of array['z'..'a'] of integer;
begin
  a[-28]['e'] := 'Y';
end.
