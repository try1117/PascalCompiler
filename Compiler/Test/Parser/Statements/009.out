Lexical exception in position (3,32) - High range limit < low range limit
